src/current/_includes/v23.1/sidebar-data/read-and-write-data.json

```diff
@@ -0,0 +1,183 @@
+{
+  "title": "Read and Write Data",
+  "is_top_level": true,
+  "items": [
+    {
+        "title": "Overview",
+        "urls": [
+          "/${VERSION}/architecture/reads-and-writes-overview.html"
+        ]
+    },
+    {
+      "title": "Write Data",
+      "items": [
+        {
+          "title": "Insert Data",
+          "urls": [
+            "/${VERSION}/insert-data.html"
+          ]
+        },
+        {
+          "title": "Update Data",
+          "urls": [
+            "/${VERSION}/update-data.html"
+          ]
+        },
+        {
+          "title": "Bulk-update Data",
+          "urls": [
+            "/${VERSION}/bulk-update-data.html"
+          ]
+        },
+        {
+          "title": "Delete Data",
+          "urls": [
+            "/${VERSION}/delete-data.html"
+          ]
+        },
+        {
+          "title": "Bulk-delete Data",
+          "urls": [
+            "/${VERSION}/bulk-delete-data.html"
+          ]
+        },
+        {
+          "title": "Batch Delete Expired Data with Row-Level TTL",
+          "urls": [
+            "/${VERSION}/row-level-ttl.html"
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Read Data",
```

Comment:
Suggest we put this section before "Write Data" so that it matches the title of the section -- "Read and Write Data"